### PR TITLE
Update Plugins documentation to include procedure to update a plugin

### PR DESCRIPTION
**Pull request by bmfmancini**

### CODE REVIEW

File: Plugins.md [hunk 1]
```diff
@@ -30,6 +30,37 @@ to enable it this can be done via clicking on the green check mark
...
+- backup the current plugin folder typically in cacti|_path/plugins/plugin_name
+
+```console
+mv /var/www/html/cacti/plugin/thold /tmp
+```
+- git clone or download the new version of the plugin
+
+```console
+git clone https://github.com/Cacti/plugin_thold.git
+```
```
Comment (netniV) [inline Plugins.md:50]: Once you have cloned a repo, you do not need to repeat the move and reclone.  You can simply use `git pull` to pull down the latest version, then `git checkout <tag>` to move to a specific version.

File: Plugins.md [hunk 2]
```diff
@@ -30,6 +30,37 @@ to enable it this can be done via clicking on the green check mark
 
 ![thold plugin enable](images/cacti_thold_enable.JPG)
 
+### Updating a Plugin
+
+Updating a plugin not only keep you up to date with the latest release of the code for that plugin
+it also allows you to keep all of your settings for that plugin ensure you DO NOT UNINSTALL the plugin
+you must instead disable the plugin which will not drop the SQL tables and thus save your settings
```
Comment (netniV) [inline Plugins.md:37]: There are some grammatical issues here (one very long sentence). Also, `keep` on the first line should really be `keeps`.

**Review by netniV — CHANGES_REQUESTED** [review]

**Comment by netniV** [comment]
Good idea for an update, needs one or two changes though.

**Comment by bmfmancini** [comment]
Hey Larry

Good point on the git pull

instead of mv plugin /tmp if you cp -R the folder to temp git pull will
work

Also we should note that if you git pull the repo you will get the absolute
latest so if you have been testing a release version it may be best to wget
the release tar

**Comment by bmfmancini** [comment]
Git pull pull the commit code for the release ? I don't usually do that I
usually wget the zip so when we are done testing in lab we move to prod
with the same updates

**Comment by bmfmancini** [comment]
@netniV  all updated let me know if this works better

**Comment by netniV** [comment]
I will have to test the git commands.  There is a way to fetch to the cache (stored in the .git folder), then checkout to a tag.  There is no need to rename folders or anything else when you do that as long you haven't modified anything.

**Comment by bmfmancini** [comment]
I also added some updates to the FAQ document

File: Plugins.md (hunk 1 above)
Comment (TheWitness) [reply Plugins.md]: I generally do a clone, in production environments.

**Comment by TheWitness** [comment]
Going to merge this.  We can fix in subsequent pulls.

**Pull request merged**